static/js/message_parser.ts

```diff
@@ -6,18 +6,21 @@ import $ from "jquery";
 // important because $("Text <a>link</a>").find("a") returns nothing;
 // one needs an outer element wrapping an object to use this
 // construction.
-function is_element_in_message_content(message, element_selector) {
+function is_element_in_message_content(
+    message: {content: string},
```

Comment:
I guess this is a workaround until we have a `Message` type for the objects in `message_store.js`?

Reply:
We probably want to at least write `type Message = {content: string}` that we can later expand and move as appropriate.

Reply:
For now, I added a type for this with the comment about moving it to `message_store`. I think we should add a temporary `types.ts` module and move this there. Another option is `global.d.ts`, however, we would need to add it to eslint globals if we want to do this.